src/Uno.UI/UI/Xaml/Controls/ScrollBar/ScrollBar.cs

```diff
@@ -969,6 +972,20 @@ internal bool IsIgnoringUserInput
 		{
 			var value = Value;
 			var change = LargeChange;
+			if (VisualTreeHelper.GetParent(VisualTreeHelper.GetParent(VisualTreeHelper.GetParent(this))) is ScrollViewer
+			    sv)
+			{
+				if (this == typeof(ScrollViewer)
+					    .GetField("_verticalScrollbar", BindingFlags.NonPublic | BindingFlags.Instance).GetValue(sv))
```

Comment:
avoid access by reflection, especially when we are in the same assembly
you can just expose it internally:
```cs
// in ScrollViewer.cs
/* pick one: */
internal ScrollBar? VerticalScrollbar { get; private set; } // replace existing field by this
internal ScrollBar? VerticalScrollbar => _verticalScrollbar; // re-expose as property
```